packages/fx-core/tests/plugins/resource/bot/unit/index.test.ts

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT license.
 import "mocha";
 import * as chai from "chai";
-import fs from "fs-extra";
+import fs = require("fs-extra");
```

Comment:
What's the difference?

Reply:
require looks like JavaScript style, and in TypeScript the recommendation is to use `import ... from`. Can you help check this?

Reply:
TsLint notice me: TS1192: Module 'fs-extra' has no default export. so, we can change that to
`import * as fs from "fs-extra";`
or use require

Reply:
Let's use `import * as fs from "fs-extra"` which align with other code